charts/flux2/values.yaml

```diff
@@ -222,3 +222,31 @@ watchallnamespaces: true
 
 # -- contents of pod imagePullSecret in form 'name=[secretName]'; applied to all controllers
 imagePullSecrets: []
+
+# -- Array of extra K8s manifests to deploy
+extraObjects: []
+  # - apiVersion: secrets-store.csi.x-k8s.io/v1
+  #   kind: SecretProviderClass
+  #   metadata:
+  #     name: fluxcd-secrets-store
+  #   spec:
+  #     provider: aws
+  #     parameters:
+  #       objects: |
+  #         - objectName: "fluxcd"
+  #           objectType: "secretsmanager"
+  #           jmesPath:
+  #               - path: "client_id"
+  #                 objectAlias: "client_id"
+  #               - path: "client_secret"
+  #                 objectAlias: "client_secret"
+  #     secretObjects:
+  #     - data:
+  #       - key: client_id
```

Comment:
Flux does not supports these key names for AWS auth. I'm guessing this example is for AWS S3?? If so the key names must be `accesskey` and `secretkey` (docs [here](https://fluxcd.io/docs/components/source/buckets/#static-authentication)).

Reply:
You are right. I just used a random example. I will adjust it, to be more flux relevant.